DuckDuckGo/AppTPActivityView.swift

```diff
@@ -25,6 +25,8 @@ struct AppTPActivityView: View {
     @ObservedObject var feedbackModel: AppTrackingProtectionFeedbackModel
     @ObservedObject var toggleViewModel = AppTPToggleViewModel()
     
+    // We only want to show "Manage Trackers" and "Report an issue" if the user has enabled AppTP at least once
+    @AppStorage("appTPUsed") var appTPUsed: Bool = false
```

Comment:
Are we able to move this into the view model and use `@UserDefaultsWrapper`? Generally we prefer to keep views as dumb as possible, and keep logic like this in the models to facilitate unit testing.